client/dvm/rpc/src/eth.rs

```diff
@@ -38,17 +38,20 @@ use sc_client_api::{
 	client::BlockchainEvents,
 };
 use sc_network::{ExHashT, NetworkService};
+use sc_transaction_pool_api::{InPoolTransaction, TransactionPool};
 use sp_api::{BlockId, Core, HeaderT, ProvideRuntimeApi};
 use sp_blockchain::{Error as BlockChainError, HeaderBackend, HeaderMetadata};
 use sp_runtime::{
 	traits::{BlakeTwo256, Block as BlockT, NumberFor, One, Saturating, UniqueSaturatedInto, Zero},
 	transaction_validity::TransactionSource,
 };
 use sp_storage::{StorageData, StorageKey};
-use sp_transaction_pool::{InPoolTransaction, TransactionPool};
 // --- std ---
 use codec::{self, Decode, Encode};
-use ethereum::{Block as EthereumBlock, Transaction as EthereumTransaction};
+use ethereum::{
+	Block as EthereumBlock, LegacyTransaction, LegacyTransactionMessage, TransactionAction,
```

Comment:
1. Recommend `BlockV0 as EthereumBlock` to replace `EthereumBlock<LegacyTransaction>`
2. `LegacyTransactionMessage` or `TransactionV0`. They are the same thing. Keep only one. Prefer the first.

Reply:
TransactionV0 is corresponding to LegacyTransaction.

I think there is no alias for LegacyTransactionMessage.